src/interoperability.md

```diff
@@ -0,0 +1,15 @@
+# Interoperability
+
+## History
+
+SORA blockchain is serve as an ecosystem for robust DeFi projects like the SORA's decentralized global economic system and [Polkaswap](./polkaswap), a DEX specifically designed for an interoperable future. In line with our vision, we recognize that blockchain maximalism is no longer viable, and we acknowledge the importance of secure, fast, and efficient asset migration between blockchains.
+
+As an initial step towards achieving this, we have developed the source code for [HASHI](./hashi), a decentralized two-way bridge connecting SORA and Ethereum. In addition to Ethereum, SORA aims to establish connections with various ecosystems. Being built on top of Substrate, it is advantageous to utilize the built-in cross-chain transfer capabilities to facilitate communication with projects running on Polkadot and Kusama ecosystems. To achieve this, the [Substrate bridge](./substrate-bridge) was developed.
+
+With multiple bridges already in operation, it becomes necessary to streamline the bridge interface to ensure consistent software components for cross-chain transfers across different heterogeneous blockchains. As a solution, the "Federated Bridge" was created to serve as the gateway between SORA mainnet and the external world. The term "federated" is used because this bridge inherits certain features from a federated architecture.
```

Comment:
```suggestion
With multiple bridges already in operation, it becomes necessary to streamline the bridge interface to ensure consistent software components for cross-chain transfers across different heterogeneous blockchains. As a solution, the "Federated Bridge" was created to serve as the gateway between the SORA mainnet and the external world. The term "federated" is used because this bridge inherits certain features from a federated architecture.
```